raytracer.ts

```diff
@@ -35,11 +36,10 @@ class Color {
     static background = Color.black;
     static defaultColor = Color.black;
     static toDrawingColor(c: Color) {
-        var legalize = d => d > 1 ? 1 : d;
         return {
-            r: Math.floor(legalize(c.r) * 255),
-            g: Math.floor(legalize(c.g) * 255),
-            b: Math.floor(legalize(c.b) * 255)
+            r: floor(min(c.r) * 255),
```

Comment:
what are these `min` doing?

Reply:
It was trying to match [the vanilla JS](https://github.com/WebReflection/raytrace/blob/27955b701bc662c87fcefe7287392bc5725d03a5/rt.vanilla.js#L34) but I missed the second argument.  I pushed a fix.